packages/server/src/api/routes/tests/application.spec.ts

```diff
@@ -178,35 +194,43 @@ describe("/applications", () => {
           path.resolve(__dirname, "data", "agency-client-portal.tar.gz")
         )
 
-      const app = await config.api.application.create({
+      const newApp = await config.api.application.create({
```

Comment:
Would it be possible to switch this to one of the new templates like `expense-approval`? Rather that depending on one we're getting rid of. Not a blocker at all!